bin/git-forgit

```diff
@@ -204,6 +205,30 @@ _forgit_stash_show() {
     return $fzf_exit_code
 }
 
+# git stash push selector
+_forgit_stash_push() {
+    _forgit_inside_work_tree || return 1
+    # Stash files if passed as arguments
+    [[ $# -ne 0 ]] && git stash push -u "$@" && return
+    local opts preview files
+    opts="
+        $FORGIT_FZF_DEFAULT_OPTS
+        -m
+        $FORGIT_STASH_PUSH_OPTS
+    "
+    preview="
+        if $_forgit_is_file_tracked; then
+            git diff --color=always {} | $_forgit_diff_pager
+        else
+            git diff --color=always /dev/null {} | $_forgit_diff_pager
+        fi
+    "
+    # Show both modified and untracked files
+    files=$(git ls-files --modified --others | FZF_DEFAULT_OPTS="$opts" fzf --preview="$preview")
```

Comment:
I think what you may want is

```
    files=$(git ls-files --exclude-standard --modified --others | FZF_DEFAULT_OPTS="$opts" fzf --preview="$preview")

```

--exclude-standard (as far as I understand) makes it so files in your .gitignore don't show up. The first time I ran this I had a ton of my build folders/object files etc show up in the output!

Reply:
Good catch! I've fixed that now.